docs/proposals/28-autonomous-shoot-clusters.md

```diff
@@ -479,6 +483,47 @@ The control plane can directly communicate with the data plane without the need
 
 There may be other components, which are affected by the changed network layout and need to be adapted.
 
+### Air-Gapped Gardener Landscapes
+
+While it is already possible today to run a Gardener landscape in an air-gapped environment, the autonomous shoot cluster feature will make it easier to set up such a landscape.
+This is due to the fact that it removes the dependency on an external Kubernetes cluster to run the Garden and act as the initial seed cluster. 
+Therefore, it may be worthwhile to take a closer look at the requirements for air-gapped Gardener landscapes and document them properly.
+
+Topics to consider (without any claim of completeness) are:
+- OCI container registry for providing container images/binaries
+- Configuration of image vector overrides to consume only local images/binaries
+- Local DNS configuration for managing Gardener landscape domains
+- TLS certificate handling, e.g. for connections between components, but also for authentication
+
+The goal would be to provide a comprehensive guide on how to set up an air-gapped Gardener landscape, which can be applied on various infrastructures.
+It may also be useful for scenarios that are not fully air-gapped, but only desire selected parts of the sovereignty this provides.
+
+### Scaling the Control Plane of Autonomous Shoot Clusters
+
+The control plane components of ordinary Gardener shoot clusters are scaled horizontally and vertically in various way.
+Autoscaling is not covered in this GEP, but it is important in the long run to ensure efficient resource usage.
+
+As far as this GEP is concerned, the control plane of autonomous shoot clusters is scaled horizontally by adding more control plane nodes (see [`gardenadm join`](#gardenadm-join)).
```

Comment:
I was thinking about this wrt autoscaling as VPA (which we heavily rely on) will not work for static pods (btw this is something that will have to be adapted as part of the implementation). 
However, we currently do not set any limits, so control plane components will just consume w/e is currently available on the control plane nodes - vertical scaling could be achieved by just using high memory nodes for the control plane. 
Additionally, we don't actually need VPA to set higher requests so that pods get scheduled on nodes with more memory.
Of coarse, this could lead to the OS killing high memory consuming pods, but this is true even today if we do not have high memory nodes on the seeds.

Reply:
I agree. The section is just meant as food for thought what could be done. At some point in time, we should look into the autoscaling topic, but it will not be the first step and it might conclude that 3 nodes are sufficient for most scenarios.